<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-PB44JCV" ignored="0" name="Testrunner_2" tests="1" failures="1" timestamp="2021-09-17T11:44:02 IST" time="120.335" errors="0">
  <testcase name="PurchaseOrder_created" time="47.336" classname="com.crm.PurchaseOrderApproved.TC_02_PurchaseOrder_created">
    <failure type="junit.framework.AssertionFailedError" message="expected:&amp;lt;false&amp;gt; but was:&amp;lt;true&amp;gt;">
      <![CDATA[junit.framework.AssertionFailedError: expected:<false> but was:<true>
at junit.framework.Assert.fail(Assert.java:50)
at junit.framework.Assert.failNotEquals(Assert.java:287)
at junit.framework.Assert.assertEquals(Assert.java:67)
at junit.framework.Assert.assertEquals(Assert.java:147)
at junit.framework.Assert.assertEquals(Assert.java:153)
at com.crm.PurchaseOrderApproved.TC_02_PurchaseOrder_created.PurchaseOrder_created(TC_02_PurchaseOrder_created.java:31)
at java.util.ArrayList.forEach(ArrayList.java:1249)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- PurchaseOrder_created -->
</testsuite> <!-- Testrunner_2 -->
